front50-web/src/main/java/com/netflix/spinnaker/front50/controllers/PipelineController.java

```diff
@@ -237,33 +258,106 @@ public Pipeline getByApplicationAndName(
   }
 
   @PreAuthorize(
-      "@fiatPermissionEvaluator.storeWholePermission() and hasPermission(#pipeline.application, 'APPLICATION', 'WRITE') and @authorizationSupport.hasRunAsUserPermission(#pipeline)")
+      "@fiatPermissionEvaluator.storeWholePermission() "
+          + "and hasPermission(#pipeline.application, 'APPLICATION', 'WRITE') "
+          + "and @authorizationSupport.hasRunAsUserPermission(#pipeline)")
   @RequestMapping(value = "", method = RequestMethod.POST)
   public synchronized Pipeline save(
       @RequestBody Pipeline pipeline,
       @RequestParam(value = "staleCheck", required = false, defaultValue = "false")
           Boolean staleCheck) {
 
-    validatePipeline(pipeline, staleCheck);
+    long saveStartTime = System.currentTimeMillis();
+    log.info(
+        "Received request to save pipeline {} in application {}",
```

Comment:
NIT:  Info?  We get a LOT of these in some cases and that'll add a lot of logs.

Reply:
As this is an important info which unambiguously confirms the pipeline save, kept it under info so that it's more readable and useful for monitoring.

Reply:
Monitoring via logs isn't great monitoring.  That's really more a category for metrics/event emission behavior.  Nominally an echo event (e.g. when using Orca TASKS this is automatically emitted via an echo event for example... since this bypasses that, that kind of thing becomes MUCH more important).

Reply:
Can we leave that for a subsequent PR?

Reply:
I'm ok with that - biggest concern is audit log behavior and NOT capturing these via normal audit logs.  This results in missed compliance possibilities.  A separate PR to make sure these events ARE auditied/sent through that would work.